tests/phpunit/includes/bootstrap.php

```diff
@@ -82,6 +82,12 @@
 	$phpunit_polyfills_autoloader = dirname( dirname( dirname( __DIR__ ) ) ) . '/vendor/yoast/phpunit-polyfills/phpunitpolyfills-autoload.php';
 	$phpunit_polyfills_error      = false;
 
+	// Define WP_TESTS_PHPUNIT_POLYFILLS_PATH if set as environment variable.
```

Comment:
If this gets added, the option to set the path via an environment variable should also be mentioned in the multi-line comment above this conditional code-block.

Reply:
Added